atlas/packages/components/src/components/RadioButton/RadioButton.tsx

```diff
@@ -10,13 +10,13 @@ export default function RadioButton({
 	label = "",
 	position = "end",
 	disabled = false,
-	onClick,
+	onClick = () => {},
 	...styleProps
 }: RadioButtonProps) {
 	const styles = useCSS({ disabled, position, ...styleProps });
 
 	return (
-		<div css={styles.container} onClick={disabled ? null : onClick}>
+		<div css={styles.container} onClick={onClick}>
```

Comment:
Why this change? We probably shouldn't call the click handler when the button is disabled (so unclickable)

Reply:
Is it a case that handler won't be called anyway because the button is disabled in the DOM?

Reply:
You're right, that was necessary, I think I had the intention to rewrite the Radio Button component but ended up skipping it in this PR. I re-added the tertiary operator and moved the rewrite to issue #836.